azure-functions-java-worker/src/main/java/com/microsoft/azure/webjobs/script/JavaWorkerClient.java

```diff
@@ -22,7 +22,11 @@
 public class JavaWorkerClient implements AutoCloseable {
     public JavaWorkerClient(IApplication app) {
         WorkerLogManager.initialize(this, app.logToConsole());
-        this.channel = ManagedChannelBuilder.forAddress(app.getHost(), app.getPort()).usePlaintext(true).build();
+        ManagedChannelBuilder<?> chanBuilder = ManagedChannelBuilder.forAddress(app.getHost(), app.getPort()).usePlaintext(true);
+        if (app.getMaxMessageSize() != null) {
+            chanBuilder.maxInboundMessageSize(app.getMaxMessageSize());
```

Comment:
For node and c# we need to set two values: max send  size, max receive size. Java probably needs maxOutboundMessageSize?

Reply:
AFAIK, Java GRPC has [unlimited default outbound message size](https://github.com/grpc/grpc-java/blob/9fac15d4f8fb09106fc4a80d761afbe1594b4d0c/core/src/main/java/io/grpc/internal/MessageFramer.java#L73).